Formula/synfig.rb

```diff
@@ -5,7 +5,7 @@ class Synfig < Formula
   mirror "https://github.com/synfig/synfig/releases/download/v1.4.2/synfig-1.4.2.tar.gz"
   sha256 "e66688b908ab2f05f87cc5a364f958a1351f101ccab3b3ade33a926453002f4e"
   license "GPL-3.0-or-later"
-  revision 1
+  revision 2
```

Comment:
Probably needs https://github.com/synfig/synfig/commit/f684b24f0db31ab8ea7aadc417fc23e3084b4138.

Will try locally to check if patch applies and there are no other issues.